src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/corpus_pruning_task_test.py

```diff
@@ -364,10 +361,10 @@ def setUp(self):
                             f'QUARANTINE_BUCKET = {self.quarantine_bucket}\n'
                             f'BACKUP_BUCKET={self.backup_bucket}\n'
                             'RELEASE_BUILD_BUCKET_PATH = '
-                            'gs://clusterfuzz-test-data/test_libfuzzer_builds/'
+                            'gs://clusterfuzz-test-data/test_libfuzzer_builds2/'
```

Comment:
test_libfuzzer_builds has a fuzzer with a crazy bug where ASAN crashes before fuzzing starts if the path name is too long (Really true!). It's not clear to me how this test was passing this whole time. I rebuilt it a functioning version in test_libfuzzer_builds2 but this one does crash on every input (which I'm not sure was the intent).
I also had another issue when I compiled this with optimizations since inlining broke some other assumptions the test makes about the result.

Reply:
Wow...